__tests__/app.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require('supertest')
+const {categoryData, commentData, reviewData, userData} = require('../db/data/test-data/index.js')
+const connection = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const app = require('../app.js')
+
+
+beforeEach(() => {return seed({categoryData, commentData, reviewData, userData})})
+afterAll(() => { return connection.end()})
+
+describe('/api/categories', () => {
+    test('GET - 200 - responds with an array of category objects', () => {
+        return request(app)
+        .get('/api/categories')
+        .expect(200)
+        .then((response) => {
+            const categories = response.body.categories
+            expect(Array.isArray(categories)).toBe(true)
+            categories.forEach((category) => {
+                expect(typeof category.slug).toBe('string')
+                expect(typeof category.description).toBe('string')
+            })
+        })
+    })
+})
+
+describe('/api', () => {
+    test('GET - 200 - responds with an JSON describing all the available endpoints on your API', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            const endpoints = response.body.endpoints
+            expect(typeof endpoints).toBe('object')
+            
+            })
+        })
+    })
+
+describe('/api/reviews/:review_id', () => {
+    test('GET - 200 - responds with a review object', () => {
+        return request(app)
+        .get('/api/reviews/3')
+        .expect(200)
+        .then((response) => {
+            const body = response.body.review
+            body.forEach((review) => {
+                expect(typeof review.votes).toBe('number')
+                expect(typeof review.title).toBe('string')
+                expect(typeof review.category).toBe('string')
+                expect(typeof review.owner).toBe('string')
+                expect(typeof review.review_id).toBe('number')
```

Comment:
There's a few properties missing on here, check the original trello ticket for more information:

should include:
review_id which is the primary key
title
review_body
designer
review_img_url
votes
category field which references the slug in the categories table
owner field that references a user's primary key (username)
created_at